### PR TITLE
Adding scripts to review

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
@petr-balogh can you also review?

File: pull_images_4.2/use_downstream_operatorresource.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# Source: Openshift-qe
+# https://github.com/openshift-qe/v3-testfiles/blob/master/logging/tools/use_downstream_operatorresource.sh#L68
+
+NAMESPACE=${1:-redhat-operators-art}
+
+
+function updateCluster()
+{
+echo "# disable redhat-operators.yaml"
+cat <<EOF >disbale_redhat_operator.yaml
```
Comment (vasukulkarni) [inline pull_images_4.2/use_downstream_operatorresource.sh]: I think we could create this as static yaml files in top level folder but not holding it for now.

**Comment by vasukulkarni** [comment]
For the follow-on PR we can arrange the structure as

/scripts
/yamls/version

and the top level folder can include main launcher, right now we can merge this as is.

**Comment by vasukulkarni** [comment]
can you sign the commits,  use "git commit -s" and it will automatically sign all commits in future.

**Comment by Shrivaibavi** [comment]
> can you sign the commits, use "git commit -s" and it will automatically sign all commits in future.

@vasukulkarni I did that

**Pull request merged**